guides/dataloader/sources.md

```diff
@@ -148,3 +148,23 @@ dataloader.with(Sources::ActiveRecordObject, Comment).merge(comments_by_id)
 ```
 
 After that, any calls to `.load(id)` will use those already-loaded records if they're available.
+
+## De-duplicating equivalent objects
+
+Sometimes, _different_ objects in the application should load the same object from `fetch`. You can customize this behavior by implementing `def result_key_for(key)` in your application. For example, to map records from your ORM to their database ID:
```

Comment:
The use-case I have is actually the opposite - two Active Record objects that point to the same DB row both `hash` and `eql?` the same, but for my loader (a version of an [`AssociationLoader`](https://github.com/Shopify/graphql-batch/blob/main/examples/association_loader.rb)) needs to see all the records  in `fetch`, without the deduping that happens when Active Record objects are used as keys in hashes/sets.

Reply:
Got it -- I updated the doc to mention this case, as well. Thanks for clarifying!